jlm/llvm/frontend/LlvmModuleConversion.cpp

```diff
@@ -49,8 +49,13 @@ convert_instructions(::llvm::Function & function, context & ctx)
   return phis;
 }
 
+/**
+ * During conversion of LLVM instructions, phi instructions become empty SsaPhiOperations.
```

Comment:
"became empty" sounds off here. More like, "During the conversion of LLVM instructions, phi instruction where created without operands. Once all...."